tests/utils/test_configurable.py

```diff
@@ -25,29 +26,6 @@ class MyOtherClass(MyBase):
 
 def test_tagged_union_deserializes():
     expected = MyClass(a=1, b="foo")
```

Comment:
Would be good to test `MyOtherClass` as well. As well as for the case where there is only one subclass